### ISSUE TITLE
evaluation set and fvd score

### ISSUE THREAD

**Issue by jikyong** [bytedance/LatentSync#348]
When running eval_fvd on the test set (an evaluation set that has been processed in the same way as the training data, with clips segmented into 5-second intervals)

However, when following the provided data_processing pipeline, the resulting clips become extremely short. At the same time, your guideline states that evaluation should be conducted on data that has undergone the same processing pipeline as the training data.

Is this understanding correct?
